modules/install/pages/upgrade-cluster-online-full-capacity.adoc

```diff
@@ -0,0 +1,236 @@
+= Upgrade a Full-Capacity, Online Cluster
+
+:description: A cluster can be upgraded while still online; and maintained at full capacity.
+:tabs:
+
+[abstract]
+{description}
+
+[#online-upgrade-with-full-capacity]
+== Understanding Full-Capacity Upgrade
+
+The context and overall requirements for upgrading a live cluster by means of the introduction of one or more additional nodes are described in xref:install:upgrade-strategies.adoc[Upgrade-Procedure Selection]: this fully explains the node-by-node upgrade of the cluster, using _swap rebalance_ to minimize overhead.
+It also explains how, if additional nodes are available, the procedure can be completed with cluster performing at full capacity, during the upgrade.
+
+The precise steps for this procedure are detailed on this page, below.
+A full understanding of the information in xref:install:upgrade-strategies.adoc[Upgrade-Procedure Selection] should be acquired, before proceeding.
+The procedure assumes that:
+
+* The cluster to be upgraded must continue to serve data throughout the cluster-upgrade process.
+
+* A single, additional node is available; to act as a _swap_ node.
```

Comment:
Yes I think it's okay here, because we are stating the assumptions for this particular example-procedure. We've mentioned "one or more" on the overview pages.